packages/airnode-utilities/src/evm/gas-prices/gas-oracle.test.ts

```diff
@@ -536,7 +536,8 @@ describe('Gas oracle', () => {
           new Promise((resolve) => {
             setTimeout(() => {
               return resolve({} as any);
-            }, GAS_ORACLE_STRATEGY_ATTEMPT_TIMEOUT_MS);
+              // Set timeout to exceed attempt maximum
+            }, GAS_ORACLE_STRATEGY_ATTEMPT_TIMEOUT_MS + 10);
```

Comment:
This is to fix a flaky test where sometimes the timeout finished within the timeout limit and the tests were not passing.